### DISCUSSION TITLE
Navigation Collections dropdown empty even though pages exist

### DISCUSSION THREAD

**Discussion by Ravenna** [statamic/cms#7387] [Troubleshooting]
I have a super simple, new Statamic install setup locally.  It's up to date and I am running on PHP 8.1.0. 

I set up a new nav and called it "Main Nav" (original, I know). But I can only create links in it, I can't link to a collection.  When I edit the navigation, there is nothing in the collections dropdown.  

I tested updating the nav yaml with:
`title: 'Main Nav'
collections: 
  - pages`

And now if I go to the nav in the cp, it gives me the option to link to an entry but if I click that the page does nothing. 

I am also seeing: 

Trying to access array offset on value of type null at /Users/tjsherrill/sites/homecourt/vendor/statamic/cms/src/Http/Controllers/CP/Fieldtypes/RelationshipFieldtypeController.php:47)

in my logs once I enabled the collection in the yaml. 

Any ideas whats up?

-- UPDATE
Well I updated to phpo 8.2 and the issue appears to be resolved. But it would be nice to sort out what happened in 8.1

**Comment by duncanmcclean** [comment]
Yeah, I believe it's a known thing (has come up a few times before). See #7117, #7263, #7316. 

Upgrading to PHP 8.2 fixes it (like you've found)

**Comment by kelenam** [comment]
Hi All,

Recently just upgraded to Statamic 4.13.2 and ran into this issue (collection entries drop-downs not populating) even with php8.2. 
I just downgraded back down to Statamic 4.13.1 and the collection entries drop-downs are working again.

This might be helpful for anyone else googling and running into the same issue or for identifying the possible root cause.

**Reply by duncanmcclean** [reply]
This sounds like it may be a bug caused in the latest release - are you able to open an issue?
